testing/jormungandr-integration-tests/src/non_functional/explorer.rs

```diff
@@ -147,6 +147,7 @@ pub fn explorer_load_test() {
         100,
         std::time::Duration::from_secs(60),
         100,
+        None,
```

Comment:
This function should be replaced by a builder API...

Reply:
Agree, It will be all refactored, but currently let's push fix for mempool